src/core/util/wire/Group.hpp

```diff
@@ -51,11 +53,22 @@ typename Group<T>::Graph Group<T>::constructGraph_(const Data& smb) {
             matIds.push_back(mat->getId());
         }
         
-        wires = lines.getMatId(matIds);
+        for (const auto& line : lines) {
+            if (
+                std::all_of(
```

Comment:
Is it possible that you actually mean any_of  ?

I think this part of the code is not very well tested. Maybe the tests are not covering this well...

Reply:
This was a really bad idea as tests from SEMBA are failing because of this. I have updated the code to iterate over the materiales and check the `matId` explicitely.